### PR TITLE
[FEATURE] Laravel 11 update

### PR DESCRIPTION
**Pull request by 77web**
Updated DocrineUserProvider and versions for some dependencies from #573.
The updates for phpunit/phpunit and doctrine/orm should be done separately.
Updated DoctrineUserProvider and Authenticable in this reposiory for Laravel 11's newly added methods in Auth interfaces.

### Changes proposed in this pull request:
- Laravel11 update

### CODE REVIEW

File: src/DoctrineServiceProvider.php [hunk 1]
```diff
@@ -88,6 +88,7 @@ protected function ensureValidatorIsUsable()
         if (!$this->isLumen()) {
             return;
         }
+        assert(property_exists($this->app, 'availableBindings'));
```
Comment (eigan) [inline src/DoctrineServiceProvider.php:91]: "added assert for lumen"

What does this mean?

Comment (77web) [reply src/DoctrineServiceProvider.php:91]: `$this->app` has "availableBindings" only when it is used in lumen.
Static analysis failed for `Illuminate\Contracts\Foundation\Application` does  not have "availableBindings" property:
`Access to an undefined property                                     Illuminate\Contracts\Foundation\Application::$availableBindings. `
https://github.com/laravel-doctrine/orm/actions/runs/8966821073/job/24623101202

The method `DoctrineServiceProvider::ensureValidatorIsUsable()` is for lumen(as shown in line 88), so the assert should be always correct here.

Comment (eigan) [reply src/DoctrineServiceProvider.php:91]: Allright, thanks 👍🏻

**Comment by eigan** [comment]
Thanks @77web! Good job 🙂

**Comment by bartjuh4** [comment]
@eigan when can we expect the new release?

**Comment by eigan** [comment]
> @eigan when can we expect the new release?

Should be out. Do composer update. Just havent written a release here om github

**Pull request merged**